src/components/views/rooms/MemberInfo.js

```diff
@@ -170,6 +171,36 @@ module.exports = React.createClass({
         this.props.onFinished();        
     },
 
+    onPowerChange: function(powerLevel) {
+        var ErrorDialog = sdk.getComponent("dialogs.ErrorDialog");
+        var roomId = this.props.member.roomId;
+        var target = this.props.member.userId;
+        var room = MatrixClientPeg.get().getRoom(roomId);
+        if (!room) {
+            this.props.onFinished();
+            return;
+        }
+        var powerLevelEvent = room.currentState.getStateEvents(
+            "m.room.power_levels", ""
+        );
+        if (!powerLevelEvent) {
+            this.props.onFinished();
+            return;
+        }
+        MatrixClientPeg.get().setPowerLevel(roomId, target, powerLevel, powerLevelEvent).done(
+        function() {
```

Comment:
Indenting nitpick: I'd probably leave the last close paren on a line by itself & indent the function so it's consistent.